FinSearchUnified/Commands/ExportCommand.php

```diff
@@ -31,7 +31,7 @@ protected function execute(InputInterface $input, OutputInterface $output)
 
         $progress->start();
         $blController->setShopKey($shopkey);
-        $blController->getFindologicXml(0, 0, true);
+        $blController->getFindologicXml('de_DE', 0, 0, true);
```

Comment:
Make the language an optional argument and default to German

Reply:
I changed this one. As the language does not affect anything in the later process. We should remove this at some point when cleaning up.